<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BmsConvertDialog</class>
 <widget class="QDialog" name="BmsConvertDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextBrowser" name="userInfoTextBrowser">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>471</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="selectBmsButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>290</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>SelectBmsFile</string>
   </property>
  </widget>
  <widget class="QPushButton" name="convertButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>290</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Convert</string>
   </property>
  </widget>
  <widget class="QListWidget" name="opernDirListWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>471</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="selectAllCheckBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Select All</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>selectAllCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>BmsConvertDialog</receiver>
   <slot>onSelectAll(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onSelectAll(int)</slot>
  <slot>onClickListItem(QListWidgetItem*)</slot>
 </slots>
</ui>
